README.md

```diff
@@ -23,10 +23,12 @@
 
 ### Requirements
 
-- [Node.js](https://nodejs.org) v8.11 or greater
+- [Node.js](https://nodejs.org) v8.11-v13.9.0
```

Comment:
node-sass does provide support all the way up to nodejs v15+; not sure the specific issue you ran into on this one, but support does vary for that package from version to version (including minor and patch releases) for sure. I would leave this as ` or greater`.

Reply:
Okay that makes sense, I just ran into a dependency issue when I've been trying to build. I had to install nodejs v13 since it had issues with my original nodejs v15. Unsure why, but I agree let's keep it as is.

Reply:
Can you revert the change back to ` or greater` please? Then I think this will be ready to go!